actionview/lib/action_view/lookup_context.rb

```diff
@@ -132,6 +132,11 @@ def exists?(name, prefixes = [], partial = false, keys = [], **options)
       end
       alias :template_exists? :exists?
 
+      def template_exists_in_any_format?(name, prefixes = [], partial = false, keys = [], **options)
+        options_with_all_formats = options.merge(formats: default_formats)
+        @view_paths.exists?(*args_for_lookup(name, prefixes, partial, keys, options_with_all_formats))
+      end
```

Comment:
Should this be private? (I don't know what the rules are for the `ViewPaths` class. If the class itself is already private then it's probably fine.

Also, can you add a comment for this method? I assume you probably know more about the differences between these methods (particularly `template_exists_in_any_format?` vs `template_exists?`), so if you can document both of them for future readers that would be great